docs/connect.html

```diff
@@ -183,6 +191,9 @@
 }
 </pre>
 
+Although not used in the example, <code>SourceTask</code> also provides two APIs to commit offsets in the source system: <code>commit</code> and <code>commitSourceRecord</code>. The APIs are provided for source systems that also need to store offsets internally which serve as an acknowledge mechanism to the source system that the message are delivered to Kafka.
```

Comment:
Second sentence is at least missing a comma (between "internally" and "which").

But I might rephrase this to make the use case a bit clearer. e.g. something like, "The APIs are provided for source systems which have an acknowledgement mechanism for messages. Overriding these methods allows the source connector to acknowledge messages in the source system, either in bulk or individually, once they have been successfully written to Kafka."